frontend/src/pages/crawls.ts

```diff
@@ -329,8 +332,15 @@ export class Crawls extends BtrixElement {
    */
   private async fetchCrawls(params?: APIPaginationQuery): Promise<void> {
     this.cancelInProgressGetCrawls();
+    window.clearTimeout(this.timerId);
+
     try {
       this.crawls = await this.getCrawls(params);
+
+      // TODO Refactor to poll task
```

Comment:
Just double-checking if this is a note for later or if you meant to do this for this PR :)

Reply:
Good catch, I'll update the comment to link to https://github.com/webrecorder/browsertrix/issues/1716